WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/WatchFaceUserSettingChangesDataSource.kt

```diff
@@ -37,7 +37,7 @@ class WatchFaceUserSettingChangesDataSource(
         val userStyleChangeListener = object : CurrentUserStyleRepository.UserStyleChangeListener {
             override fun onUserStyleChanged(userStyle: UserStyle) {
                 Log.d(TAG, "onUserStyleChanged(): userStyle: $userStyle")
-                trySend(userStyle)
+                trySend(UserStyle(userStyle))
```

Comment:
I believe this should no longer be necessary with `alpha20`, which included the changes to equals/hashcode for `UserStyle`.

Reply:
Ha, good call, just saw alpha 20 was released today... I will revise.